postsubmit.yaml

```diff
@@ -39,12 +39,12 @@ steps:
   - 'CLOUDSDK_COMPUTE_ZONE=europe-west4-a'
   - 'CLOUDSDK_CONTAINER_CLUSTER=xl-ml-test-europe-west4'
 - name: 'gcr.io/cloud-builders/kubectl'
-  args: ['apply', '-f', 'k8s/us-central1/', '-f', 'k8s/common/',]
+  args: ['apply', '-f', 'k8s/us-central1/gcs-buckets.yaml', '-f', 'k8s/us-central1/volumes.yaml', '-f', 'k8s/common/',]
   env:
   - 'CLOUDSDK_COMPUTE_ZONE=us-central1-b'
   - 'CLOUDSDK_CONTAINER_CLUSTER=oneshots'
 - name: 'gcr.io/cloud-builders/kubectl'
-  args: ['apply', '-f', 'k8s/europe-west4/', '-f', 'k8s/common/',]
+  args: ['apply', '-f', 'k8s/europe-west4/gcs-buckets.yaml', '-f', 'k8s/common/',]
```

Comment:
I'm confused here and on line 42: are these changes related to the new `profile-cleanup.yaml` file? It's fine to add these changes even if they're unrelated. Just wanted to make sure you didn't mean to add `profile-cleanup.yaml` to the postsubmit script

Reply:
It's related. The idea is to keep the profile cleanup job from being created once in the main cluster and once on the oneshot cluster. I'm specifying only the files that need to be created on oneshots.